src/simulation/m_global_parameters.fpp

```diff
@@ -670,12 +710,37 @@ contains
             integral(i)%ymax = dflt_real
         end do
 
-        ! GRCBC flags
-        #:for dir in {'x', 'y', 'z'}
-            bc_${dir}$%grcbc_in = .false.
-            bc_${dir}$%grcbc_out = .false.
-            bc_${dir}$%grcbc_vel_out = .false.
-        #:endfor
```

Comment:
Were these lines added back somewhere else? I'm not seeing them anywhere

Reply:
My apologies for the oversight. Those lines were unintentionally removed, and I am now adding them back.

Reply:
Fixed